File: Plugins/Aspose_Pdf_Java_for_PHP/src/Aspose/Pdf/WorkingWithDocumentObject/RemoveMetadata.php
<?php

namespace Aspose\Pdf\WorkingWithDocumentObject;

use com\aspose\pdf\Document as Document;

class RemoveMetadata {

    public static function run($dataDir=null)
    {

        # Open a pdf document.
        $doc = new Document($dataDir . "input1.pdf");

        if (preg_match('/pdfaid:part/',$doc->getMetadata())) {
            $doc->getMetadata()->removeItem("pdfaid:part");
        }

        if (preg_match('/dc:format/',$doc->getMetadata())) {
            $doc->getMetadata()->removeItem("dc:format");
        }

        # save update document with new information
        $doc->save($dataDir . "Remove_Metadata.pdf");

        print "Removed metadata successfully, please check output file." . PHP_EOL;

    }
}

File: plugins/Aspose_Pdf_Java_for_Ruby/lib/asposepdfjava/Document/removemetadata.rb
module Asposepdfjava
  module RemoveMetadata
    def run
    	# The path to the documents directory.
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
        
        # Open a pdf document.
        doc = Asposepdfjava.java_class('com.aspose.pdf.Document').new(data_dir + "input1.pdf")

        if doc.getMetadata().contains("pdfaid:part")
            doc.getMetadata().removeItem("pdfaid:part")
        end    
        
        if doc.getMetadata().contains("dc:format")
            doc.getMetadata().removeItem("dc:format")
        end

        # save update document with new information
        doc.save(data_dir + "Remove_Metadata.pdf")

        puts "Removed metadata successfully, please check output file."
    end
  end
end
